create-projects/create-cd-jenkins.sh

```diff
@@ -1,80 +1,82 @@
 #!/usr/bin/env bash
-set -eux
+set -eu
 
 # This script sets up the cd/jenkins-master and the associated webhook proxy.
 
 # support pointing to patched tailor using TAILOR environment variable
-: ${TAILOR:=tailor}
-
-tailor_exe=$(type -P ${TAILOR})
-tailor_version=$(${TAILOR} version)
-
-echo "Using tailor ${tailor_version} from ${tailor_exe}"
 
+TAILOR="tailor"
 DEBUG=false
 STATUS=false
-while [[ $# -gt 0 ]]
-do
-key="$1"
-
-case $key in
-    --status)
-    STATUS=true
-    ;;
-    -d|--debug)
-    DEBUG=true;
-    shift
-    ;;
-    *)
-        echo "Unknown option: $1. Exiting."
-        exit 1
-    ;;
-esac
-shift # past argument or value
-done
+FORCE=""
+
+function usage {
+   printf "usage: %s [options]\n", $0
+   printf "\t--status\tExecutes tailor status\n"
+   printf "\t--force\tIgnores warnings and error with tailor --force\n"
+   printf "\t-h|--help\tPrints the usage\n"
+   printf "\t-d|--debug\tEnabled debug\n"
+   printf "\t-t|--tailor\tChanges the executable of tailor. Default: tailor\n"
+
+}
+
 
+while [[ "$#" -gt 0 ]]; do case $1 in
+   --status) STATUS=true;;
+   -d|--debug) DEBUG=true; set -x;;
```

Comment:
I like it, but that was the default previously. Maybe we add it to the arguments in the Jenkinsfile? Also, I would prefer this to be called `--verbose` to align with Tailor

Reply:
Done!